nextflow.config

```diff
@@ -416,12 +416,14 @@ params {
     }
 
     mlst {
-        singularity = "https://depot.galaxyproject.org/singularity/mlst%3A2.23.0--hdfd78af_1"
-        docker = "quay.io/biocontainers/mlst:2.23.0--hdfd78af_1"
+        version = "2.23.0"
+        singularity = "https://depot.galaxyproject.org/singularity/mlst%3A${version}--hdfd78af_1"
+        docker = "quay.io/biocontainers/mlst:${version}--hdfd78af_1"
```

Comment:
Could you remove the `quay.io` prefix here since otherwise the container will not be cached in our container registry. This must have been missed from before.